src/pudl/transform/ferc1.py

```diff
@@ -6636,28 +6636,42 @@ def make_calculation_dimensions_explicit(
 ) -> pd.DataFrame:
     """Fill in null dimensions w/ the values observed in :func:`table_dimensions_ferc1`.
 
-    In the raw XBRL metadata's calculations, there is an implicit assumption that the
-    calculations are valid within a set of shared axises/primary key values. We have
-    mannually added some calculations which span multiple tables where the calculation
-    is only applicable within a subset of these axis/primary key columns. For example,
-    the :ref:`utility_plant_summary_ferc1` table includes many different
-    ``utility_types``, but factoids from that table are calculable from factoids in
-    the :ref:`plant_in_service_ferc1` table - which only includes the ``utilty_type`` of
-    ``electric``.
-
-    This function fills in all of the non-specified dimensions with the values found in
-    :func:`table_dimensions_ferc1`. This is often a broadcast merge because many tables
-    contain many values within these dimension columns.
-
-    Any dimension that was specified will not be touched. Any remaining nulls are a
-    result of never having observed values for that ``table_name`` and
-    ``xbrl_factoid``.
+    In the raw XBRL metadata's calculations, there is an implicit assumption that
+    calculated values are aggregated within categorical columns called Axes or
+    dimensions, in addition to being grouped by date, utility, table, and fact. However,
+    the relevant dimensions and values are not specified in the calculation components,
+    and the same calculation is assumed to apply to all groups within a dimension.
+
+    We have extended this calculation system to allow independent calculations to be
+    specified for different values within a given dimension, and for these calculations
+    to include values reported in multiple tables.  For example, the
```

Comment:
`and for these calculations to include values reported in multiple tables.`

This is not new or super duper relevant imo.

Reply:
I was adapting a sentence that already existed talking about calculating spanning multiple tables. I can remove it.